README.md

```diff
@@ -14,11 +14,18 @@ support Clojure (JVM).
 
 [//]: # (This should be kept in sync with docs/tutorial.md)
 
+The following commands can either be executed directly if you have the
+dependencies on your host. Otherwise, you can use the following
+command to launch a "dev" container with the build and runtime dependencies:
```

Comment:
I know we mention conlink in the Tests section, but this is now the first time it would be used in the README.  We might want to mention here that Open vSwitch is required on the host to use the docker-compose files.

Reply:
For the specific build task here at the top of the README, only the "dev" container is being started (run), so conlink isn't used for that.

However, thinking about this more, I realized that most uses of conlink don't actually need ovs (including clj-protocol). So we probably we want this PR to conlink https://github.com/LonoCloud/conlink/pull/13 to merge first and then I'll update this one.

I'm working on a separate PR for clj-protocol that will add a github action workflow that will build on this one and does the docker-compose+conlink tests too (not just the build and unit tests that are currently run).

Reply:
Oops, good call on the `run`. I missed that.  Thanks for adding "auto"!